styles.css

```diff
@@ -0,0 +1,113 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: montserrat;
+    color: #fff;
+}
+
+/* **Header** */
+header {
+    background-color: #46424D;
+    height: 60px;
+}
+h1 {
+    padding: 1% 0;
+    color: #fff;
+    font-size: 1.5em;
+    font-weight: 300;
+    text-align: center;
+}
+main {
+    background-color: #2b2733;
+    height: 95vh;
+}
+h2 {
+width: 60vw;
+margin: 30px 30px 15px 30px;
+font-size: 24px;
```

Comment:
I think we should stay consistent with units in sizing font. Change this line from `px` to `em`?